test/unit/cuda/CMakeLists.txt

```diff
@@ -21,13 +21,13 @@ raja_add_test(
   NAME test-cuda-reduce-loc
   SOURCES test-reduce-loc.cpp)
 
-raja_add_test(
-  NAME test-cuda-reduce-tupleloc
-  SOURCES test-reduce-tupleloc.cpp)
+#raja_add_test(
```

Comment:
Should these be commented?

Reply:
Shoot. Thanks for catching that. Change pushed with comments removed.

Reply:
For now, yes they ought to commented out. Those tests are breaking on CUDA8+clang5 that Argonne is using. The tests compile on all of our builds, and pass at runtime.

Reply:
Since they pass all of our builds, they should not be commented out.